crowdbotics/modules
# PLAT-9778 add rn web build to deployment pipeline

## Ticket

[PLAT-9778](https://crowdbotics.atlassian.net/browse/PLAT-9778)
_Related tickets:_
_Related PRs:_

## Type of PR


## Changes introduced
Add Dockerfile on project root to include RN build pipeline
Some update  to webpack config to use django `{%.static %}` tag for rn build file

Had to add the Dockerfile to root, since heroku doesn't allow to customize docker Build Context and just use the directory containing Dockerfile as the build context.

## Test and review

1. Create a mobile app
2. Deploy API platform, it should deploy the rn web build
